1_fetch.R

```diff
@@ -36,10 +36,14 @@ p1_targets_list <- list(
     priority = 0.99 # default priority (0.8) is set globally in _targets.R
   ),
   
-  # Load harmonized WQP data product for discrete samples
+  # Load harmonized WQP data product for discrete samples. Note that this data
+  # frame was created from an (internal) DRB data harmonization GitLab repository:
+  # https://code.usgs.gov/wma/iws/wq-data-harmonization/-/commit/b705a2b91877d5e0bb9386250659b46149295b55
+  # The output rds file was downloaded from the inland salinity sharepoint folder
+  # and manually placed in 1_fetch/in.
   tar_target(
     p1_wqp_data_rds,
-    "1_fetch/in/DRB.WQdata.rds",
+    "1_fetch/in/20220731_DRB_WQdata.rds",
```

Comment:
I updated this target to pull from ScienceBase when we were using the original WQdata file from Shoda et al. 2019 (in [this](https://github.com/USGS-R/drb-inland-salinity-ml/pull/180/commits/ef2579683170e33fcab37564de01e05d4f0f61aa) commit). I did this just to add that pattern to our commit history, even though I ultimately revert back to this rds import pattern. 

The updated discrete salinity dataset that Christine Wieben and I generated is not yet on ScienceBase. I envision updating this target when/if that data gets pushed to SB. I don't think I'll have time to create an SB data release for the WQ data this year, so it might be a good idea to ask Meg and Christine for their thoughts on that.